### PR TITLE
Fix broken query selector

### PR DESCRIPTION
**Pull request by jridgewell**
Fixes https://github.com/ampproject/amphtml/issues/30822

### CODE REVIEW

File: build-system/eslint-rules/query-selector.js [hunk 1]
```diff
@@ -157,26 +177,35 @@ module.exports = function (context) {
         });
       }
 
-      selector = quasis.join('');
+      selector = accumulator + quasis[quasis.length - 1];
```
Comment (samouri) [inline build-system/eslint-rules/query-selector.js:180]: was this causing issues?

**Review by samouri — APPROVED** [review]
thanks for fixing this. I'm excited to remove the whole path once we update the InOb polyfill to understand rootMargin in x-origin-iframe and cleanup the intersect-resources experiment.

File: build-system/eslint-rules/query-selector.js (hunk 1 above)
Comment (jridgewell) [reply build-system/eslint-rules/query-selector.js:180]: Without this, it was ignoring the final static portion of the template literal. Eg, `foo${expression}bar` would leave out `bar`.

**Review by jridgewell — COMMENTED** [review]
I don't think this can be removed with either of those experiments. We'd have to keep this until we remove IE11 support.

**Comment by samouri** [comment]
You're right! I had thought the intrinsic fix was below the InOb gate, but its actually right before it 😬.
https://github.com/ampproject/amphtml/blob/dd35e0768ec92c6cbe70460a9380cf301d1a8cf4/src/service/resources-impl.js#L356-L363

**Comment by erwinmombay** [comment]
per @jridgewell, request to force merge.

**Comment by erwinmombay** [comment]
everything is green except bundle size but its minor

**Pull request merged**